### PR TITLE
Update free-courses-bn.md

### PR DESCRIPTION
**Pull request by NeamulIslamFahim**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-bn.md [hunk 1]
```diff
@@ -144,6 +144,7 @@ Shikkhangon BD
 * [Java Swing Bangla Tutorials](https://www.youtube.com/playlist?list=PLgH5QX0i9K3rAHKr6IteF5kdgN6BorH9l) - Anisul Islam
 * [জাভা এন্টারপ্রাইজ এডিশন](https://dimikcomputing.com/course/javaee-online-course/) - দ্বিমিক কম্পিউটিং
 * [Java Bangla (বাংলা) tutorial for beginners](https://youtube.com/playlist?list=PL82MewGFQkx1jjozz7I98Yjanw8n6p9HP) - Time & Training
+* [Java Course! Full Course Bangla](https://youtube.com/playlist?list=PLysy6TYSMvpJmMN0ElbC936RmmFustDiy) - Online Course
```
Comment (shrn01) [inline courses/free-courses-bn.md:147]: alphabetically order the resources

**Review by NeamulIslamFahim — COMMENTED** [review]
Please check

**Comment by eshellman** [comment]
look at linter results - it still wants spacing fixed

**Review by NeamulIslamFahim — COMMENTED** [review]
Please check

File: courses/free-courses-bn.md [hunk 2]
```diff
@@ -141,11 +141,13 @@ Shikkhangon BD
 ### Java
 
 * [Java Bangla Tutorials \| CORE Java \| Complete OOP](https://www.youtube.com/playlist?list=PLgH5QX0i9K3oAZUB2QXR-dZac0c9HNyRa) - Anisul Islam
+* [Java Course!Full Course Bangla](https://www.youtube.com/playlist?list=PLysy6TYSMvpJmMN0ElbC936RmmFustDiy) - Online Course
 * [Java Swing Bangla Tutorials](https://www.youtube.com/playlist?list=PLgH5QX0i9K3rAHKr6IteF5kdgN6BorH9l) - Anisul Islam
 * [জাভা এন্টারপ্রাইজ এডিশন](https://dimikcomputing.com/course/javaee-online-course/) - দ্বিমিক কম্পিউটিং
 * [Java Bangla (বাংলা) tutorial for beginners](https://youtube.com/playlist?list=PL82MewGFQkx1jjozz7I98Yjanw8n6p9HP) - Time & Training
 
 
+
```
Comment (shrn01) [inline courses/free-courses-bn.md:150]:
```suggestion
```

Only 2 lines between sections

**Review by shrn01 — CHANGES_REQUESTED** [review]

**Review by NeamulIslamFahim — COMMENTED** [review]
Is it ok?

File: courses/free-courses-bn.md [hunk 3]
```diff
@@ -141,6 +141,7 @@ Shikkhangon BD
 ### Java
 
 * [Java Bangla Tutorials \| CORE Java \| Complete OOP](https://www.youtube.com/playlist?list=PLgH5QX0i9K3oAZUB2QXR-dZac0c9HNyRa) - Anisul Islam
+* [Java Course!Full Course Bangla](https://www.youtube.com/playlist?list=PLysy6TYSMvpJmMN0ElbC936RmmFustDiy) - Online Course
```
Comment (shrn01) [inline courses/free-courses-bn.md:144]:
```suggestion
* [Java Course! Full Course Bangla](https://www.youtube.com/playlist?list=PLysy6TYSMvpJmMN0ElbC936RmmFustDiy) - Online Course
```

**Review by NeamulIslamFahim — COMMENTED** [review]
Now check.
Thanks a lot for helping and apology for the errors

**Pull request merged**